lib/moonshot/change_set.rb

```diff
@@ -76,7 +76,10 @@ def delete
     # NOTE: At the time of this patch, AWS-SDK native Waiters do not
     # have support for ChangeSets. Once they add this, we can make
     # this code much better.
+    # Still no support for this waiter, but it's planned.
+    # https://github.com/aws/aws-sdk-ruby/issues/1388
     def wait_for_change_set
+      wait_seconds = Moonshot.config.changeset_wait_time || 30
```

Comment:
Makes sense.  I see it in the 20+ second range from time to time.  I feel like we should cover at least the 95% case here so 60 or 90 seconds seems reasonable.